src/app/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -228,14 +228,18 @@ export class CreateEditNewsComponent extends FormBaseComponent implements OnInit
       });
   }
 
+  filterArr = (item, index) => [
+    ...this.filters.slice(0, index),
+    item,
+    ...this.filters.slice(index + 1)
+  ]
+
```

Comment:
let's be consistent and use 1 format for methods
private filterArray (item: TYPEHERE, index: TYPEHERE) {
 return [
    ...this.filters.slice(0, index),
    item,
    ...this.filters.slice(index + 1)
  ]
  }

Reply:
requested changes has been added with this commit 5b599444b94ca7e421d3fb03f6986cd4c07bb879
+ types for the argument
+ public data modifiers
- es6 short-hand syntax has been removed